Detections/MultipleDataSources/ZincJan272021IOCs.yaml

```diff
@@ -0,0 +1,120 @@
+id: 09551db0-e147-4a0c-9e7b-918f88847605
+name: Known ZINC Comebacker and Klackring malware hashes
+description: |
+  'ZINC attacks against security researcher campaign malware hashes.'
+severity: High
+requiredDataConnectors:
+  - connectorId: DNS
+    dataTypes:
+      - DnsEvents
+  - connectorId: AzureMonitor(VMInsights) 
+    dataTypes:
+      - VMConnection
+  - connectorId: CiscoASA
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: PaloAltoNetworks
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - DeviceProcessEvents
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - DeviceFileEvents
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - DeviceNetworkEvents
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandAndControl
```

Comment:
T1204 is part of the Execution tactic, should CommandAndControl be here?  If so, that Technique is associated with it